.github/workflows/main.yml

```diff
@@ -145,11 +145,144 @@ jobs:
         run: |
           python jenkins_scripts/registry.py build cpp_backends --dir docker --username "$DOCKER_USERNAME" --branch "$GIT_BRANCH"  --registry "$DOCKER_REGISTRY"
           python jenkins_scripts/registry.py push cpp_backends --username "$DOCKER_USERNAME" --registry "$DOCKER_REGISTRY"
+      - run: true
+
+  publish_images_terragraph:
```

Comment:
This is the same as the publish_images job except $DOCKER_USERNAME is replaced with $GITHUB_USERNAME - instead of duplicating the job, could we conditionally some vars like REGISTRY_USERNAME based on whether this on the TGNMS repo or not?